src/test/attachto.ts

```diff
@@ -1,12 +1,13 @@
-var assert = require('assert');
-var snabbdom = require('../snabbdom');
+import assert from 'assert'
+import { init } from '../snabbdom'
+import { RemoveHook } from '../hooks';
 
-var patch = snabbdom.init([]);
-var attachTo = require('../helpers/attachto').default;
-var h = require('../h').default;
+var patch = init([]);
+import attachTo from '../helpers/attachto'
+import h from '../h'
 
 describe('attachTo', function() {
-  var elm, vnode0;
+  var elm: any, vnode0: any;
```

Comment:
This pattern of reusing these two variables is in all test modules. These definitely should be refactored, I feel. Probably removed and replaced with variables/constants in each test. But beyond the scope of this PR.

Reply:
Open a new issue?